rtc-video-quality/generate_data.py

```diff
@@ -372,9 +372,9 @@ def decode_file(job, temp_dir, encoded_file):
   with open(os.devnull, 'w') as devnull:
     if job['codec'] in ['av1', 'vp8', 'vp9']:
       decoder = 'aom/aomdec' if job['codec'] == 'av1' else 'libvpx/vpxdec'
-      subprocess.check_call([decoder, '--i420', '--codec=%s' % job['codec'], '-o', decoded_file, encoded_file, '--framestats=%s' % framestats_file], stdout=devnull, stderr=devnull)
+      subprocess.check_call([decoder, '--i420', '--codec=%s' % job['codec'], '-o', decoded_file, encoded_file, '--framestats=%s' % framestats_file], stdout=devnull, stderr=devnull, encoding='utf-8')
```

Comment:
This line is too long. The Google Python style guide requires lines to be limited to 80 characters. We don't necessarily need to do that, but lines should be short enough that they don't get cut off when viewed in GitHub without scrolling horizontally.
Please split this line, and any other already long lines which you have made longer.

Reply:
Sorry, I added this comment without starting a review before, and then deleted it to make it part of the review, but I see you already replied to the one I deleted. I'll try not to do that again. Still getting used to reviews on github.

Reply:
This will be updated in the next pull request

Reply:
Okay, sounds good.